src/js/view/view.js

```diff
@@ -953,11 +918,9 @@ define([
             // player display
             switch (state) {
                 case states.PLAYING:
+                    _userActivity();
```

Comment:
This should be in the _isAudioFile case below. State changing to play should not result in controls being shown as if there was user activity.